src/nft/hooks/useBag.ts

```diff
@@ -79,7 +82,11 @@ export const useBag = create<BagState>()(
       addAssetToBag: (asset) =>
```

Comment:
since the `fromSweep` is optional and by default is false, you should be able to remove this now overridden function definition `addAssetToBag: (asset: UpdatedGenieAsset) => void`

Reply:
going to change everything to use the bulk addAssetsToBag